src/IO/DGF.cpp

```diff
@@ -107,7 +107,7 @@ return; }
     \param[in] filename dgf file name
 */
 DGFObj::DGFObj(
-    std::string                          filename
+    const std::string                   &filename
```

Comment:
If I well understood the modifications (from copy value to reference) are not related to the title of the commit.

Reply:
In my view, "misusages" include also the cases where references should have been used but they were not (i.e., all the "*: pass large types by constant reference" commits).

Reply:
The title of the commit says "IO/DGF: pass small and trivially-copyable types by value". am I wrong ?

Reply:
I though you meant the title of the pull request. I will fix the commit message.